q2_fondue/scraper.py

```diff
@@ -218,31 +218,32 @@ def _find_special_id(txt: str, pattern: str, split_str: str) -> list:
 
 
 def _find_accession_ids(txt: str, id_type: str) -> list:
-    """Returns list of run, study, BioProject and other IDs found in `txt`.
+    """Returns list of run, study, BioProject, experiment and
+    sample IDs found in `txt`.
 
-    Searching for these patterns of accession IDs that are currently
+    Searching for these patterns of accession IDs that are all also
     supported by other q2fondue actions:
     BioProject ID: PRJ(E|D|N)[A-Z][0-9]+
     Study ID: (E|D|S)RP[0-9]{6,}
     Run ID: (E|D|S)RR[0-9]{6,}
-    Also sample and experiment IDs are fetched grouped as 'other' IDs
-    as the q2fondue actions do not support them yet:
     Experiment ID: (E|D|S)RX[0-9]{6,}
     Sample ID: (E|D|S)RS[0-9]{6,}
```

Comment:
Experiment and Sample IDs take 7 numeric digits, right? Not an essential detail since the actual regex takes any number, just checking for documentation's sake.

Reply:
I have seen 6 and 7 digit experiment and sample IDs. The way I understand `{6,}` is number of digits >= 6 (as depicted [here](https://stackoverflow.com/a/9652050) or [here under "limiting repetition"](https://www.regular-expressions.info/repeat.html)).

Reply:
Oh good to know! The only ones I'd seen were 7 digits, so that sounds good 👍